hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ArtifactManagementTest.java

```diff
@@ -173,14 +173,70 @@ public void createArtifactsUntilQuotaIsExceeded() throws NoSuchAlgorithmExceptio
                 .isEqualTo(maxArtifacts - 1);
 
         // now we should be able to create an artifact again
-        final byte random[] = RandomStringUtils.random(artifactSize).getBytes();
-        try (final InputStream inputStream = new ByteArrayInputStream(random)) {
+        final byte[] randomBytes = randomBytes(artifactSize);
+        try (final InputStream inputStream = new ByteArrayInputStream(randomBytes)) {
             artifactManagement.create(inputStream, sm1.getId(), "fileXYZ", false, artifactSize);
             assertThat(artifactRepository.findBySoftwareModuleId(PAGE, sm1.getId()).getTotalElements())
                     .isEqualTo(maxArtifacts);
         }
     }
 
+    @Test
+    @Description("Verifies that the quota specifying the maximum artifact storage is enforced (across software modules).")
+    public void createArtifactsUntilStorageQuotaIsExceeded() throws NoSuchAlgorithmException, IOException {
+
+        // create as many artifacts as possible w/o violating the storage quota
+        final long maxBytes = quotaManagement.getMaxArtifactStorage();
+        final List<Long> artifactIds = Lists.newArrayList();
+        final int artifactSize = 320 * 1024;
```

Comment:
Could you write the comment why the size is 320 kb, or better reverse the logic and specify the number of artifacts __numArtifacts__ and calculate the __artifactSize__. In my opinion, it would look a little clearer.

Reply:
Done